### PR TITLE
Feature/add repositories details

### PR DESCRIPTION
**Pull request by brunoparma88**
* A functionality was created in repositories, which lists all the repositories of each git graph. Similar to the department feature.
* Minimal changes in scss styles

### CODE REVIEW

File: app/assets/images/polygon 2.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="9" height="12" viewBox="0 0 9 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M9 6L-4.89399e-07 11.1962L-3.51373e-08 0.803847L9 6Z" fill="#5F85FF"/>
```
Comment (sebastiancaraballo) [inline app/assets/images/polygon 2.svg:2]: Check the name of the file.

**Review by sebastiancaraballo — COMMENTED** [review]
please add tests

File: app/views/pull_requests/time_to_merge_prs_repository/index.erb [hunk 2]
```diff
@@ -0,0 +1,49 @@
+
+
```
Comment (enzofab91) [inline app/views/pull_requests/time_to_merge_prs_repository/index.erb:2]: Remove these empty lines

File: app/views/pull_requests/time_to_second_review_prs_repository/index.erb [hunk 3]
```diff
@@ -0,0 +1,48 @@
+
```
Comment (enzofab91) [inline app/views/pull_requests/time_to_second_review_prs_repository/index.erb:1]: Same as above.

**Review by enzofab91 — APPROVED** [review]
Nice! Left some minor comments.

File: app/views/development_metrics/merge_time/_success_rate_detail.erb [hunk 4]
```diff
@@ -2,7 +2,7 @@
   <p class='alert alert-secondary'>
     Out of <%= merge_time_success_rate[:total] %> pull requests,
     <%= merge_time_success_rate[:successful] %> were merged in the first
-    <%= merge_time_success_rate[:metric_detail][:value] %> hours.
+    <%= merge_time_success_rate[:metric_detail][:value] %> hours.1
```
[the remark this answers is not in the document]
Comment (brunoparma88) [reply app/views/development_metrics/merge_time/_success_rate_detail.erb:5]: its not ok

**Review by sebastiancaraballo — APPROVED** [review]

File: app/views/development_metrics/_products_list 2.erb [hunk 5]
```diff
@@ -0,0 +1,25 @@
+<div class="list-group">
```
Comment (RobertoCarrilloAvila) [inline app/views/development_metrics/_products_list 2.erb:1]: Check the name of the file

File: app/views/shared/_pull_request_list.erb [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+      <div class="metrics-container container col-md-12">
+        <div class="metrics shadow-box p-3">
+
+        <% pull_requests.each do |range,pr| %>
+          <h6><%= range %> hours:</h6>
+          <ul>
+            <% pr.each do |pr_link| %>
+              <li>
+                <%= link_to nil, pr_link, target: '_blank' %>
+                </li>
```
Comment (RobertoCarrilloAvila) [inline app/views/shared/_pull_request_list.erb:14]: the closing tag is not indented correctly

File: app/controllers/pull_requests/pull_requests_controller.rb [hunk 7]
```diff
@@ -0,0 +1,6 @@
+module PullRequests
+  class PullRequestsController < ApplicationController
+    layout 'sidebar_metrics'
+    include LoadSettings
```
Comment (martinjaimem) [inline app/controllers/pull_requests/pull_requests_controller.rb:4]:
```suggestion
    include LoadSettings
    
    def index
      @pull_requests = repository.call(
        from: metric_params[:period],
        repository_name: params[:repository_name]
      )
    end

    def metric_params
      params.require(:metric).permit(:period)
    end
```

File: app/controllers/pull_requests/time_to_merge_prs_repository_controller.rb [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+    def index
+      @pull_requests = Builders::Distribution::PullRequests::TimeToMergeRepository.call(
+        from: metric_params[:period],
+        repository_name: params[:repository_name]
+      )
+    end
+
+    def metric_params
+      params.require(:metric).permit(:period)
+    end
```
Comment (martinjaimem) [inline app/controllers/pull_requests/time_to_merge_prs_repository_controller.rb:12]:
```suggestion
    def repository
      Builders::Distribution::PullRequests::TimeToMergeRepository
    end
```

File: app/controllers/pull_requests/time_to_second_review_prs_repository_controller.rb [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+      @pull_requests = Builders::Distribution::PullRequests::TimeToSecondReviewRepository
+                       .call(
+                         from: metric_params[:period],
+                         repository_name: params[:repository_name]
+                       )
+    end
+
+    def metric_params
+      params.require(:metric).permit(:period)
+    end
```
Comment (martinjaimem) [inline app/controllers/pull_requests/time_to_second_review_prs_repository_controller.rb:13]:
```suggestion
    def repository
      Builders::Distribution::PullRequests::TimeToSecondReviewRepository
    end
```

File: spec/requests/pull_requests/time_to_second_review_prs_repository/index_spec.rb [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+      {
+        metric: {
+          period: 4
+        }
+      }
+    end
+
+    before do
+      subject
+    end
```
Comment (martinjaimem) [inline spec/requests/pull_requests/time_to_second_review_prs_repository/index_spec.rb:78]:
```suggestion
    before { subject }
```

File: spec/services/builders/distribution/pull_requests/time_to_merge_repository_spec.rb [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+          repository_name: repository_ok.name,
+          from: 4
+        )
+      end
+
+      before do
+        first_pull_request.update!(merged_at: Time.zone.now)
+        second_pull_request.update!(merged_at: Time.zone.now)
+        third_pull_request.update!(merged_at: Time.zone.now)
+      end
```
Comment (martinjaimem) [inline spec/services/builders/distribution/pull_requests/time_to_merge_repository_spec.rb:43]: why not doing this in the declaration of the `let`s?

File: spec/services/builders/distribution/pull_requests/time_to_second_review_repository_spec.rb [hunk 12]
```diff
@@ -0,0 +1,100 @@
+require 'rails_helper'
+
+RSpec.describe Builders::Distribution::PullRequests::TimeToSecondReviewRepository do
+  describe '.call' do
+    let(:user1) { create(:user, login: 'user1') }
+    let(:user2) { create(:user, login: 'user2') }
+    let(:user3) { create(:user, login: 'user3') }
+
+    let(:repository_ok) { create(:repository) }
```
Comment (martinjaimem) [inline spec/services/builders/distribution/pull_requests/time_to_second_review_repository_spec.rb:9]: what does "ok" mean here? can we think of another naming for this to make it more clear? 🤔

File: app/views/shared/_pull_request_list.erb [hunk 13]
```diff
@@ -0,0 +1,22 @@
...
+            <% end %>
+          </ul>
+        <% end %>
+        </div>
+    </div>
+  </div>
+</div>
+</div>
```
Comment (martinjaimem) [inline app/views/shared/_pull_request_list.erb:22]: we should ident the html 👍

File: spec/requests/pull_requests/time_to_merge_prs_repository/index_spec.rb [hunk 14]
```diff
@@ -0,0 +1,65 @@
...
+
+    it 'renders pull requests index view' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'returns pull requests variable with data' do
+      expect(assigns(:pull_requests)).not_to be_empty
+    end
+
+    it 'returns correct repositories' do
```
Comment (martinjaimem) [inline spec/requests/pull_requests/time_to_merge_prs_repository/index_spec.rb:60]: Maybe we should try re-naming the descriptions of the tests.
The reason behind that is that, as an example here, if I read the test, it says:
```
  Time To Merge PRs Repository, With correct params, returns correct repositories
```

So, the reader doesn't really know what the page should render, right?
I mean, what do we consider "correct"?

File: app/views/shared/_pull_request_list.erb [hunk 15]
```diff
@@ -0,0 +1,22 @@
+<div class="container-fluid">
+  <div class="row p-3">
+
+      <div class="row p-3">
```
Comment (martinjaimem) [inline app/views/shared/_pull_request_list.erb:4]: we should be consistent and remove all empty lines in the HTML 👍

**Comment by martinjaimem** [comment]
Goof work 🚀 , left some minor comments 👍

**Review by martinjaimem — APPROVED** [review]

**Pull request merged**